en/dev_setup/fast-dds-installation.md

```diff
@@ -0,0 +1,162 @@
+# Fast DDS Installation
+
+<img alt="logo" src="../../assets/fastrtps/eprosima_logo.png" style="float:left;"/> [eProsima Fast DDS](https://github.com/eProsima/Fast-DDS) is a C++ implementation of the Object Management Group's (OMG) Data Distribution Service (DDS) specification and the Real Time Publish Subscribe (RTPS) protocol.
```

Comment:
FYI, I heavily subedited this introduction and moved some of the additional points to the end. I didn't think we actually need to "fully" describe the DDS/RTPS relationship here other than to say that you need this for ROS2 - i.e. what people actually will care about.  Worth a sanity check though.

Reply:
All good!